dependencydiff/raw_dependencies.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dependencydiff
+
+import "github.com/ossf/scorecard/v4/pkg"
+
+// Dependency is a raw dependency fetched from the GitHub Dependency Review API.
+type Dependency struct {
```

Comment:
How about implementing `FetchDependencyDiffData()` in this file?
This way we can hide `Dependency` and make it private. This function can populate the `DependencyCheckResult`